src-tauri/src/transport/commands.rs

```diff
@@ -9,7 +9,7 @@ use serde::{Deserialize, Serialize};
 use tauri::ipc::InvokeBody;
 use tauri::{
     command,
-    ipc::{Request},
+    ipc::Request,
```

Comment:
Let's remove this change, since it's not necessarily part of the parameter display change